run.ts

```diff
@@ -71,18 +70,18 @@ export async function runWithBrowser(
 }
 
 export interface RunWithDenoOptions {
-  results: (readonly [fileName: string, exitCode: number])[]
+  results: (readonly [filePath: string, exitCode: number])[]
   reloadUrl?: string
   signal: AbortSignal
 }
 
 export async function runWithDeno({ reloadUrl, results, signal }: RunWithDenoOptions) {
-  return (async (dir: string, fileName: string) => {
-    const result = await executionWrapper(fileName, async (outputBuffer) => {
+  return (async (filePath: string) => {
+    const result = await executionWrapper(filePath, async (outputBuffer) => {
       const command = new Deno.Command(Deno.execPath(), {
         args: ["run", "-A"]
           .concat(reloadUrl ? [`-r=${reloadUrl}`] : [])
-          .concat([`${dir}/${fileName}`]),
+          .concat([filePath]),
```

Comment:
Why use `concat` instead of passing `filePath` directly into the last position of the `args` list?

Reply:
because reloadUrl has to go before the `filePath` and its optional so it can't be inlined in args

Reply:
... not sure if I fully understand. Could it be destructured (the ternary) in place?

Reply:
` args: ["run", "-A", reloadUrl ? `-r=${reloadUrl}` : "", filePath],`

if i do something like this i get this if i dont pass in the reload url

```
finished examples/dynamic.eg.ts
running examples/read/now.eg.ts
examples/read/now.eg.ts failed -- console output:
error: Is a directory (os error 21)
```

Reply:
```diff
- args: ["run", "-A", reloadUrl ? `-r=${reloadUrl}` : "", filePath],
- args: ["run", "-A", ...reloadUrl ? [`-r=${reloadUrl}`] : [], filePath],
```

Reply:
:open_mouth: